lib/shiftzilla/release.rb

```diff
@@ -5,7 +5,7 @@ class Release
     attr_reader :name, :targets, :milestones, :default, :token
 
     def initialize(release,builtin=false)
-      @name       = release['name']
+      @name       = release['name'].to_s
```

Comment:
@thrasher-redhat updated PR to force release names to String class. Protects against 'X.Y' version names from being handled as Floats.